### PR TITLE
added location tile

### PR DESCRIPTION
**Pull request by FearsomeRover**
I added the location tile, but noticed after, that the number of tiles is different on this dev branch, than it is on the main one. 12 or 13 organizers? If its 13, we need to modify the layout a little imo.

### CODE REVIEW

File: src/components/tiles/location-tile.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { Tile } from './tile';
+
+const mapLinks = {
+  google: 'https://goo.gl/maps/dkXw1AjgX351WY4J6',
+  waze: 'https://ul.waze.com/ul?preview_venue_id=12517851.124916363.422156&navigate=yes',
+};
+
+export function LocationTile() {
+  return (
+    <Tile className='col-span-3 min-h-[500px]'>
```
Comment (Tschonti) [inline src/components/tiles/location-tile.tsx]: This has to be `sm:col-span-2 lg:col-span-3` 
This is the class of the container: `grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3` So there is 1 column on base width, 2 on small, 3 above lg. If an item takes up more columns than there are, it messes up the other items

**Review by Tschonti — CHANGES_REQUESTED** [review]
There are 13 organisers, Beni is missing from staging for some reason. But I think the layout can stay, this wouldn't really fit in 2/3 of the screen.

Also, I found one edge case where the layout is messed up:
image

Not a huge issue, if you can't solve it in a few minutes, you can leave it xd

**Review by Tschonti — APPROVED** [review]
👌

**Comment by FearsomeRover** [comment]
image

**Pull request merged**